### ISSUE TITLE
"message" event is not raised when payload is empty

### ISSUE THREAD

**Issue by ZachBray** [kaazing/javascript.client#13]
In our system we have some messages that only contain headers. This works fine when only using RabbitMQ. However, in the Kaazing AMQP JavaScript API `AmqpChannel.onmessage` is not called when the payload of a message is empty.

**Comment by sbadugu** [comment]
@dpwspoon @cmebarrow Should we investigate this ?

**Comment by dpwspoon** [comment]
Yes, this seems like a bug.  I would expect that we have the same bug in 4.x client so I'm assigning to 5.0 + 4.x milestone.  Hopefully we can get this fixed for that release and forward port here.  @sanjay-saxena or @chao-sun-kaazing do one of you want to take this?
